src/common/ManagedCommon/ColorFormatHelper.cs

```diff
@@ -452,8 +455,9 @@ private static string GetStringRepresentation(Color color, char paramFormat, str
                     return ((color.R * 65536) + (color.G * 256) + color.B).ToString(CultureInfo.InvariantCulture);
                 case "Dv":
                     return (color.R + (color.G * 256) + (color.B * 65536)).ToString(CultureInfo.InvariantCulture);
-                case "Na":
-                    return ColorNameHelper.GetColorName(color);
+
+                // case "Na":
+                //     return ColorNameHelper.GetColorName(color);
```

Comment:
I guess this needs to be treated differently because each project that uses it actually needs to get into their own resources to get the name, right? Please add some comments to explain the situation, since this is an exception to what's happening. That will help reading this code.